frame/pablo/src/lib.rs

```diff
@@ -238,6 +262,25 @@ pub mod pallet {
 
 		#[pallet::constant]
 		type PalletId: Get<PalletId>;
+
+		// Why this is
```

Comment:
Required for the spot price to be valid, as the ratio might lead to 0 before normalization